naked-objects/LICENSE

```diff
@@ -0,0 +1,201 @@
+ Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
Due to this license, we need to update the `LICENSE.md` at the root of the project to state that `naked-objects` is licensed under Apache